core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/DataSource.scala

```diff
@@ -19,6 +19,10 @@ object DataSource {
   }
 }
 
+object AttributeTempTableType extends Enumeration {
```

Comment:
Are there ever times where we will want to use temp tables for both? For example, in `SubmissionMonitorActor`, we call `handleOutputs`, which can write to both workspace and entity attributes.

Reply:
Yes, I think the code should do the right thing in that case.  `handleOuputs` is wrapped by  `datasource.inTransactionWithAttrTempTable` which should create both temp tables assuming `UncoordinatedDataSourceAccess` is used.  Is that a good assumption?

Reply:
Ah! Okay - I commented with only a partial understanding with what was going on! Sorry

Reply:
No worries at all!